packages/styles/src/mixins/button/_button-types.scss

```diff
@@ -233,6 +233,14 @@ $button-types-config: (
   @include _fd-button-type($default-button-config);
 
   @each $type, $props in $button-types-config {
+
+    &--#{$type},
+    &--#{$type} &__text {
+      @if $type == "emphasized" {
```

Comment:
@InnaAtanasova what do you think about that?

Reply:
looks good